mailpoet/assets/js/src/automation/automation.tsx

```diff
@@ -1,36 +1,30 @@
 import ReactDOM from 'react-dom';
+import { Workflow } from './listing/workflow';
+import { AutomationListing } from './listing';
+import { Onboarding } from './onboarding';
+import { Loading } from '../common';
```

Comment:
I don't think we should use the MailPoet-specific spinner here. There are some spinners in `@wordpress/components` or maybe in the Woo components but I think I'll just revert it and leave it for later.